PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3555,17 +3570,23 @@ struct HfTrackIndexSkimCreatorLfCascades {
               std::array<float, 3> pVecPion1XiHyp = {0.};
               df2.getTrack(0).getPxPyPzGlo(pVecXi);
               df2.getTrack(1).getPxPyPzGlo(pVecPion1XiHyp);
+              float p_T_Xic = std::hypot(pVecXi[0] + pVecPion1XiHyp[0], pVecXi[1] + pVecPion1XiHyp[1]);
```

Comment:
```suggestion
              float ptXic = RecoDecay::pt(RecoDecay::pVec(pVecXi, pVecPion1XiHyp));
```

Reply:
`float ptXic = RecoDecay::pt(pVecXi, pVecPion1XiHyp);`

Reply:
Dear experts, which one is better?